pyproject.toml

```diff
@@ -76,10 +68,6 @@ exclude = [
 
   [tool.poetry.extras]
   docs = [
```

Comment:
The _docs_ extras group should contain the documentation's dependencies, given the dependencies will be set as optional, the only way to install them would be to include the _docs_ extras group.